core/src/main/scala/chisel3/Printable.scala

```diff
@@ -155,12 +159,16 @@ case class Printables(pables: Iterable[Printable]) extends Printable {
     val (fmts, args) = pables.map(_.unpack(ctx)).unzip
     (fmts.mkString, args.flatten)
   }
+
+  final def unpack_args(): Seq[Bits] = pables.map(_.unpack_args()).flatten.toSeq
```

Comment:
```suggestion
  final def unpackArgs: Seq[Bits] = pables.view.flatMap(_.unpackArgs).toList
```
`.view` will ensure we don't create intermediate collections, `.flatMap` == `.map` + `.flatten`, and we should pick concrete types. These are typically small (and never massive) so `List` is best.